### PR TITLE
Add BananaPi BPI-PicoW-S3 .

### PR DESCRIPTION
**Pull request by Wind-stormger**
Adds support for the BananaPi BPI-PicoW-S3 Boards.
Same size as Raspberry Pi Pico board.
Based on esp32s3 chip, 8M flash, 4-layer PCB, half-hole process.
With one WS2812 LED, one monochrome LED, one ceramic antenna. 
Support double-reset to tinyUF2.
A pair of contacts is reserved for triggering BOOT0 after short circuit, which is usually not used.
PID from Espressif.

### CODE REVIEW

**Comment by Wind-stormger** [comment]
[tinyuf2/pull/250](https://github.com/adafruit/tinyuf2/pull/250)

File: ports/espressif/boards/bpi_picow_s3/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+CFLAGS += -DCFG_TUD_TASK_QUEUE_SZ=32
+
+CIRCUITPY_ESP_FLASH_MODE = dio
+CIRCUITPY_ESP_FLASH_FREQ = 80m
+CIRCUITPY_ESP_FLASH_SIZE = 8MB
+
+#CIRCUITPY_BITBANG_NEOPIXEL = 1
+
+# Include these Python libraries in firmware.
+FROZEN_MPY_DIRS += $(TOP)/frozen/Adafruit_CircuitPython_NeoPixel
```
Comment (dhalbert) [inline ports/espressif/boards/bpi_picow_s3/mpconfigboard.mk:22]: Are you freezing this for convenience? Usually we don't freeze libraries unless there are space reasons.

File: ports/espressif/boards/bpi_picow_s3/mpconfigboard.mk [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# The default queue depth of 16 overflows on release builds,
+# so increase it to 32.
+CFLAGS += -DCFG_TUD_TASK_QUEUE_SZ=32
+
+CIRCUITPY_ESP_FLASH_MODE = dio
+CIRCUITPY_ESP_FLASH_FREQ = 80m
+CIRCUITPY_ESP_FLASH_SIZE = 8MB
+
+#CIRCUITPY_BITBANG_NEOPIXEL = 1
+
```
Comment (dhalbert) [inline ports/espressif/boards/bpi_picow_s3/mpconfigboard.mk:20]: Just remove instead of commenting out.
```suggestion
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
Hi, thanks for the board! I had a couple of questions.

**Comment by Wind-stormger** [comment]
https://github.com/adafruit/circuitpython-org/pull/1070

File: ports/espressif/boards/bpi_picow_s3/mpconfigboard.mk (hunk 1 above)
Comment (Wind-stormger) [reply ports/espressif/boards/bpi_picow_s3/mpconfigboard.mk:22]: Yes, this is very convenient indeed.
Can I be allowed to freeze it?

File: ports/espressif/boards/bpi_picow_s3/mpconfigboard.mk (hunk 2 above)
Comment (Wind-stormger) [reply ports/espressif/boards/bpi_picow_s3/mpconfigboard.mk]: Thanks, removed.

File: ports/espressif/boards/bpi_picow_s3/mpconfigboard.mk (hunk 1 above)
Comment (dhalbert) [reply ports/espressif/boards/bpi_picow_s3/mpconfigboard.mk:20]: It's your board, so sure.

**Review by dhalbert — APPROVED** [review]
All set!

**Pull request merged**